iroha_python/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "iroha_python"
+version = "0.1.0"
+authors = ["Iroha 2 team <https://github.com/orgs/soramitsu/teams/iroha2>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[lib]
+name = "iroha2_sys"
```

Comment:
Maybe `iroha2_bindings`?

Reply:
I kinda like naming with `_sys`, as it is very rusty